### ISSUE TITLE
add pickaxe

### ISSUE THREAD

**Issue by zachmylack** [studio-minus/ppg-bugs-and-requests#1145]
it a spawnable melee weapon that can pierce some living things (like the human or gorse) and can hit into wood or the boulder or whatever

i wanted it when i saw the boulder was made of minerals

i made my own with a stell rod and wood but it looked teribble

**Comment by DindinYT37** [comment]
Please use the template.

**Comment by Drogo-man** [comment]
> Please use the template. 

He did, he just also removed the titles.

**Comment by DindinYT37** [comment]
> > Please use the template.
> 
> He did, he just also removed the titles.

Yeah, but the template is not only to provide the right information but also for us to read them easier.

**Comment by ppgplayer** [comment]
That is nasty and sadistic. Make it happen zooi.
